patches/tModLoader/Terraria/ModLoader/Core/ModOrganizer.cs

```diff
@@ -727,6 +727,19 @@ internal static void CleanupOldPublish(string repo)
 		return information;
 	}
 
+	internal static List<ModVersionHash> GetModHashesFromFolder(string folderPath)
```

Comment:
I think this was better in `WorkshopSocialModule`, I don't see this function being relevant for the general operation of mod loading and launching